src/views/EventInitiation.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="page-container">
+    <NavBtns
+      :topType="'share-dark'"
+      :botType="currStep === 0 ? 'next-only' : 'default'"
+      @next="nextStep()"
+      @back="prevStep()"
+    />
+
+    <!-- step 1 -->
+    <div class="step-1 step" v-if="currStep === 0">
+      <img src="../assets/icons/event-initiation/hourglass-icon.svg" alt class="step-icon" />
+      <h1 class="step-title">Date / Time</h1>
+      <div class="date-wrapper">
+        <v-menu
+          ref="calendarMenu"
+          max-width="290"
+          :close-on-content-click="false"
+          offset-y
+          :return-value.sync="eventDate"
+        >
+          <template v-slot:activator="{ on }">
+            <textarea
+              type="text"
+              v-on="on"
+              placeholder="When should the event be held?"
+              class="step-input"
+              v-model="eventDate"
+            ></textarea>
+          </template>
+          <v-date-picker v-model="eventDate">
+            <v-btn text color="primary" @click="showCalendar = false">Cancel</v-btn>
+            <v-spacer></v-spacer>
+            <v-btn
+              color="primary"
+              @click="
+                $refs.calendarMenu.save(eventDate);
+                showDatePicker = false;
+                showTimePicker = true;
+              "
+            >Save</v-btn>
+          </v-date-picker>
+          <!-- <v-time-picker v-model="eventTime">
+            <v-btn text color="primary" @click="showTimePicker = false"
+              >Cancel</v-btn
+            >
+            <v-spacer></v-spacer>
+            <v-btn color="primary" @click="$refs.calendarMenu.save(eventTime)"
+              >Save</v-btn
+            >
+          </v-time-picker>-->
+        </v-menu>
+      </div>
+      <div class="media">
+        <img src="../assets/vectors/date-vector.svg" class="step-vector" />
+      </div>
+      <div class="step-options">
+        <div class="checkbox">
+          <v-checkbox label v-model="establishedEvent" value="value" color="primary"></v-checkbox>
```

Comment:
On my laptop this checkbox is covering the 'next' button so I cant click it 😢

Reply:
Same when the page is on 'step 3'. Image is covering the buttons 
<img width="510" alt="Screen Shot 2020-02-25 at 3 10 33 AM" src="https://user-images.githubusercontent.com/44561376/75242255-95dbe980-577c-11ea-8db9-4b331524ba1a.png">

Reply:
hmm i'll look into this, i tried making it responsive already but i guess your screen is even smaller.